iothub/service/src/ExportImportDevice.cs

```diff
@@ -11,27 +11,51 @@
 namespace Microsoft.Azure.Devices
 {
     /// <summary>
-    ///  contains device properties specified during export/import operation
+    /// Contains device properties specified during export/import operation
     /// </summary>
     public sealed class ExportImportDevice
     {
         private string _eTag;
         private string _twinETag;
 
         /// <summary>
-        /// making default ctor public
+        /// Property container
+        /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1034:Nested types should not be visible", Justification = "Publicly property. No behavior changes allowed.")]
```

Comment:
Why did the class get moved?

Reply:
I just moved it to the top of the wrapper class.